### ISSUE TITLE
No idea how to contact you except through a github issue-

### ISSUE THREAD

**Issue by Lukalot** [whjochim/xwacomcalibrate#1]
I wanted to say you have solved my month long frustration trying to map my Wacom tablet to monitors when needed and even more so mapping it to individual windows when necessary. Thank you so much!

**Comment by whjochim** [comment]
Thank you for the nice words, and I'm glad this could help someone!
This project was highly specific to my needs, because my monitor is an unusual aspect ratio.

Independent of the distribution you are actually using, you get the best advice in the archlinux wiki!
https://wiki.archlinux.org/title/wacom_tablet#Adjusting_aspect_ratios

I'll take this Issue as a hint that there is demand for it, and I'll look into rewriting it a bit!

**Comment by Lukalot** [comment]
Yeah, unfortunately I mostly read forum posts which guided me in the wrong direction.

I set to watch this repo, if you do update anything that would be great. Either way, thanks a lot!
